swift/Plugins/CompileSlice/CompileSlicePlugin.swift

```diff
@@ -6,9 +6,13 @@ import Foundation
 // The entry point for command line builds with SPM
 extension CompileSlicePlugin: BuildToolPlugin {
     func createBuildCommands(context: PluginContext, target: Target) async throws -> [Command] {
+        guard let swiftTarget = target as? SwiftSourceModuleTarget else {
+            throw PluginError.invalidTarget(target)
```

Comment:
I removed this on man. We'll need to add it back but pass the stringified version of target as Target is not Sendable.